packages/ui-kit/src/components/PieChart/PieChart.types.ts

```diff
@@ -0,0 +1,63 @@
+import type { ChartConfiguration } from 'chart.js'
+import { DimensionInput, Sort } from 'src/helpers'
+import { ErrorFallbackProps } from '../ErrorFallback'
+import { DataComponentProps, QueryProps } from '../shared.types'
+
+type PieChartVariant = 'pie' | 'doughnut'
+
+export type ChartProps = {
+  /** Sets the position of the labels */
+  labelPosition?: 'axis' | 'inside' | 'top'
+
+  /** Hides the legend on chart If it is set the true
+   * @default false
+   */
+  isLegendHidden?: boolean
```

Comment:
Is it more ergonomic to have `showLegend` or `hideLegend`? @hasancolak @felipecadavid @sashathor 

```suggestion
  hideLegend?: boolean
```

Reply:
I'd prefer to use `hideLegend`, it makes sense to show it by default then you just need to pass the prop to hide it